apps/status/lib/status.ex

```diff
@@ -70,12 +70,9 @@ defmodule Status do
 
   @spec is_enabled?() :: boolean() | nil
   defp is_enabled?() do
-    case {Application.get_env(:status, :metrics), Normalize.to_boolean(System.get_env("METRICS"))} do
-      {_, true} -> true
-      {_, false} -> false
-      {true, _} -> true
-      {false, _} -> false
-      _ -> nil
-    end
+    Normalize.to_boolean(System.get_env("METRICS"))
+  rescue
+    Normalize.ToBooleanError ->
+      Application.get_env(:status, :metrics)
```

Comment:
Hmm, I don't really see the need for this. How about something like:

```
  defp is_enabled?() do
    :status
    |> Application.get_env(:metrics)
    |> Normalize.to_boolean(()
  end
```

And having the `metrics` value set in the config files:

prod
```
config :status,
  metrics: {:system, "METRICS"}
```

I'm not sure if this up should have `false` as default as well, the whole thing stays optional, right? 

dev
```
config :status,
  metrics: {:system, "METRICS", false}
```

test
```
config :status,
  metrics: {:system, "METRICS", false}
```

I think this is the pattern we follow for most of our ENVs.

Reply:
Could also just put it in the main config file if we use `false` as default everywhere.

Reply:
I've noticed that (at least Phoenix) is moving away from this approach: https://github.com/phoenixframework/phoenix/blob/70606bf9e98b0a838ccea1d5e376d4dec78c446b/lib/phoenix/socket/transport.ex#L506

Does this apply here?

Reply:
Hmm, I think we went that way because of Deferred Config (I could be wrong) 🤔 If we decide to move away from this, it should be done globally throughout the project. For now, let's keep it consistent. Thanks for raising it tho, didn't know Phoenix was moving away from it.